include/lbann/layers/layer.hpp

```diff
@@ -866,9 +870,11 @@ class Layer
   /// @todo Remove
 
   SubGraphCommunication subgraph_communication_method = PT2PT;
-
+  //model-level sub-graph parallelism
   bool apply_subgraph_parallelism = false;
-
+  //Layer-level sub-graph execution
+  bool m_subgraph_parallelism_execution = false;
+  //Process-level sub-graph execution
```

Comment:
What does "Process-level" mean in this context?

Reply:
Model-level: Is subgraph parallelism enabled for Model?
Layer-level: Does this layer need subgraph execution (like split and sum layers)
Process-level: Will this layer run in the given process (For e.g. some layers will only on a subset of processes defined by grid tag)

Reply:
1. when you say "enabled for Model" do you mean "enabled for *this* Model?"
2. the name "process-level execution" for "is this executed on this process" may be unclear with respect to the others
3. Please add the explanations to the code itself as comments so it's clear for a reader and not just me

thanks!